[2024-09-09T18:51:10.756-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-09T18:51:10.770-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_dbt_dag.dag_dbt_raw_model manual__2024-09-09T21:50:55.400080+00:00 [queued]>
[2024-09-09T18:51:10.779-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_dbt_dag.dag_dbt_raw_model manual__2024-09-09T21:50:55.400080+00:00 [queued]>
[2024-09-09T18:51:10.780-0300] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-09T18:51:10.810-0300] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): dag_dbt_raw_model> on 2024-09-09 21:50:55.400080+00:00
[2024-09-09T18:51:10.814-0300] {standard_task_runner.py:72} INFO - Started process 186091 to run task
[2024-09-09T18:51:10.816-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dbt_dag', 'dag_dbt_raw_model', 'manual__2024-09-09T21:50:55.400080+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt_raw_model.py', '--cfg-path', '/tmp/tmphv8q5c_0']
[2024-09-09T18:51:10.818-0300] {standard_task_runner.py:105} INFO - Job 82: Subtask dag_dbt_raw_model
[2024-09-09T18:51:10.877-0300] {task_command.py:467} INFO - Running <TaskInstance: simple_dbt_dag.dag_dbt_raw_model manual__2024-09-09T21:50:55.400080+00:00 [running]> on host DESKTOP-0CT35O1.
[2024-09-09T18:51:10.961-0300] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matheus' AIRFLOW_CTX_DAG_ID='simple_dbt_dag' AIRFLOW_CTX_TASK_ID='dag_dbt_raw_model' AIRFLOW_CTX_EXECUTION_DATE='2024-09-09T21:50:55.400080+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-09T21:50:55.400080+00:00'
[2024-09-09T18:51:10.963-0300] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-09T18:51:10.965-0300] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-09T18:51:10.966-0300] {logging_mixin.py:190} INFO - Current task name:dag_dbt_raw_model state:running start_date:2024-09-09 21:51:10.771564+00:00
[2024-09-09T18:51:10.967-0300] {logging_mixin.py:190} INFO - Dag name:simple_dbt_dag and current dag run status:running
[2024-09-09T18:51:10.968-0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-09T18:51:10.970-0300] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-09T18:51:10.972-0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "pyenv activate dbt_env && cd '/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/dbt_project/bancos_airflow' && dbt run --models bancos_raw"]
[2024-09-09T18:51:10.977-0300] {subprocess.py:86} INFO - Output:
[2024-09-09T18:51:11.007-0300] {subprocess.py:93} INFO - /usr/bin/bash: line 1: pyenv: command not found
[2024-09-09T18:51:11.008-0300] {subprocess.py:97} INFO - Command exited with return code 127
[2024-09-09T18:51:11.018-0300] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/operators/bash.py", line 249, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2024-09-09T18:51:11.037-0300] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-09-09T18:51:11.038-0300] {logging_mixin.py:190} INFO - Task start:2024-09-09 21:51:10.771564+00:00 end:2024-09-09 21:51:11.037285+00:00 duration:0.265721
[2024-09-09T18:51:11.040-0300] {logging_mixin.py:190} INFO - Task:<Task(BashOperator): dag_dbt_raw_model> dag:<DAG: simple_dbt_dag> dagrun:<DagRun simple_dbt_dag @ 2024-09-09 21:50:55.400080+00:00: manual__2024-09-09T21:50:55.400080+00:00, state:running, queued_at: 2024-09-09 21:50:55.442474+00:00. externally triggered: True>
[2024-09-09T18:51:11.041-0300] {logging_mixin.py:190} INFO - Failure caused by Bash command failed. The command returned a non-zero exit code 127.
[2024-09-09T18:51:11.042-0300] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=simple_dbt_dag, task_id=dag_dbt_raw_model, run_id=manual__2024-09-09T21:50:55.400080+00:00, execution_date=20240909T215055, start_date=20240909T215110, end_date=20240909T215111
[2024-09-09T18:51:11.073-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-09T18:51:11.074-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 82 for task dag_dbt_raw_model (Bash command failed. The command returned a non-zero exit code 127.; 186091)
Traceback (most recent call last):
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/operators/bash.py", line 249, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2024-09-09T18:51:11.111-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-09-09T18:51:11.125-0300] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-09T18:51:11.127-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
